### ISSUE TITLE
Migrate to SvelteKit 2

### ISSUE THREAD

**Issue by HendrikSchmidt** [digitalservicebund/achill#21]
With the [release of SvelteKit 2](https://svelte.dev/blog/sveltekit-2), it might be sensible to [migrate the code](https://kit.svelte.dev/docs/migrating-to-sveltekit-2) to support [Vite 5](https://vitejs.dev/blog/announcing-vite5) and pave the way for Svelte 5.

However, with the existing efforts to [migrate to Remix](https://github.com/digitalservicebund/achill/tree/remix-migration), I am unsure whether Svelte(Kit) is the choice going forward. Maybe you can weigh in, @malte-laukoetter?

**Comment by malte-laukoetter** [comment]
We actually first thought about upgrading the Svelte & SvelteKit version instead of migrating to remix. The project is currently using a really old version of SvelteKit. We first tried updating SvelteKit to the current version. This is sadly not that straight forward as there is no upgrade guide or even a good list of things that changed. Also when starting to upgrade it, it became clear quite quickly that a lot of things changed and that this would effectively be almost a complete rewrite.
We then decided that it would be better to migrate to Remix than to a newer version of SvelteKit.

So currently there is no plan to continue the work using Svelte. Instead the plan is to finish the migration to remix (and the personio integration) and then merge that branch.
